### PR TITLE
CFINSPEC-367 Added option for silent inspec parallel run

### PR DESCRIPTION
**Pull request by Nik08**
## Description

Added option for silent inspec parallel run. Now it can be done with `--ui silent`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: lib/plugins/inspec-parallel/lib/inspec-parallel/super_reporter/silent.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module InspecPlugins::Parallelism
+  class SuperReporter
+    class Silent < InspecPlugins::Parallelism::SuperReporter::Base
+    end
+  end
+end
```
Comment (clintoncwolfe) [inline lib/plugins/inspec-parallel/lib/inspec-parallel/super_reporter/silent.rb]: Zen master Nikita!

**Review by clintoncwolfe — APPROVED** [review]
This is the sound of one hand clapping.

File: lib/plugins/inspec-parallel/lib/inspec-parallel/super_reporter/silent.rb (hunk 1 above)
Comment (Nik08) [reply lib/plugins/inspec-parallel/lib/inspec-parallel/super_reporter/silent.rb]: Haha! 🧘🏻

**Comment by Vasu1105** [comment]
As a reviewer, I felt that the empty class definition should have code comments so that the person who doesn't know why it's created has more clarity with it while reviewing the PR.

**Review by ahasunos — APPROVED** [review]

**Pull request merged**